cmd/tools/generate/generate.go

```diff
@@ -250,6 +250,18 @@ func generateDocker(path string, kind int) {
 	}
 	_, _ = fmt.Fprintf(os.Stderr, "Wrote file_sd targets to %s\n", opts.filesdPath)
 
+	fmt.Println(os.Getenv("HARVEST_DOCKER"))
+	if os.Getenv("HARVEST_DOCKER") == "yes" {
```

Comment:
we discussed using `os.Getenv("HARVEST_DOCKER") != ""` on Friday to more closely match LookupEnv, but I'm fine with either